### ISSUE TITLE
Install/Configure database management tools

### ISSUE THREAD

**Issue by felixalbrigtsen** [Programvareverkstedet/Drift#46]
Regarding our new database sever, we will need some way for members to manage and use their databases.
The primary use for this is to allow users without root access to create their own databases.
- [mysql-admtools](https://dev.pvv.ntnu.no/projects/mysql-admtools/browser#trunk/bin) ([Used at NTNU](https://i.ntnu.no/wiki/-/wiki/English/Using+MySQL+at+NTNU), allows creation of databases and users, but not data )

We could also host clients for databases, but this is not required for operation, as users can run their own clients locally.

- [phpmyadmin](https://www.phpmyadmin.net/) (web interface for managing MySQL databases and data, but not users)
- [adminer](https://www.adminer.org/) Minimal alternative to phpmyadmin
- [pgadmin](https://www.pgadmin.org/) Web interface for managing PHP databases

**Comment by felixalbrigtsen** [comment]
This is on hold for now. Users may contact drift@pvv.ntnu.no or #drift in our Matrix-server to request new databases.
The management software has found to be insecure. The code must be fixed, and a NixOS-package must be written before this can be enabled on our current database server.

**Comment by h7x4** [comment]
Depends on #64
